apps/src/code-studio/components/libraries/LibraryClientApi.js

```diff
@@ -22,7 +22,8 @@ export default class LibraryClientApi {
           this.cacheBustSuffix = new Date().getTime();
           onSuccess(data);
         }
-      }
+      },
+      {includeDetails: true}
```

Comment:
This is a new `options` argument to the `put` function in the `clientApi`: https://github.com/code-dot-org/code-dot-org/pull/51983/files#diff-8d5f8d9c671cdfda3c972c2575a96df3c07eb331d05cfb45e169a4f7aadf9b38L193